### ISSUE TITLE
Toolbar colors in light mode in Spyder 5

### ISSUE THREAD

**Issue by gb119** [spyder-ide/spyder#15326]
### Issue Report Checklist

* [X] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [X] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [X] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [X] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [X] Restarted Spyder
    * [X] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [X] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

There are some issues with toolbar colours when using Spyder 5.0.x in light mode:
a) The toolbar background is inconsistent between panes and the main toolbar and sometimes within a pane (e.g. find pane):

![image](https://user-images.githubusercontent.com/4428426/115120907-d3ff2280-9fa7-11eb-88d2-5f6acd2f139c.png)

b) The contrast on the buttons is much worse than in Spyder 4 (and I can't see how to change it) - this can be an accessibility issue as well as just being a little irritating... :-) cf. Spyder 4 below:

![image](https://user-images.githubusercontent.com/4428426/115120939-0b6dcf00-9fa8-11eb-8a4e-bf21f59048f2.png)

Dark mode is better for point (a) but still has issues for point (b). 5.0.1 overall much better than 5.0.0 through) :-)

### What steps reproduce the problem?

1. Switch Spyder to light mode in Preferences Appearance tab and restart (Spyder 3 icon set)
2. Compare with Spyder 4

### What is the expected output? What do you see instead?

I would have expected a consistent background colour to the toolbars in the panes and the main toolbar (and certainly not have buttons with a different background to the toolbar!). I would like to have a better contrast, or someway to adjust the contrast.

## Versions

* Spyder version:  5.0.1
* Python version:  3.7.9
* Qt version:  4.9.7
* PyQt version:  4.9.2
* Operating System name/version:  Win 10 x64

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.19.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  21.8.0 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.7.2 (OK)
pygments >=2.0                :  2.7.3 (OK)
pylint >=1.0                  :  2.6.0 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.4 (OK)
setuptools >=39.0.0           :  51.0.0.post20201207 (OK)
sphinx >=0.6.6                :  3.4.2 (OK)
spyder_kernels >=2.0.1;<2.1.0 :  2.0.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  0.10.4 (OK)
zmq >=17                      :  20.0.0 (OK)

# Optional:
cython >=0.21                 :  0.29.21 (OK)
matplotlib >=2.0.0            :  3.3.2 (OK)
numpy >=1.7                   :  1.19.2 (OK)
pandas >=1.1.1                :  1.2.0 (OK)
scipy >=0.17.0                :  1.5.2 (OK)
sympy >=0.7.3                 :  1.7.1 (OK)

```

**Comment by gb119** [comment]
Ok, realize that the button background is indicating state, so perhaps the underlying problem is the extent of the difference in background gray level between toolbar/on state and the pane/off state. I'm afraid it just looks ugly to my eyes in light mode! A less dark gray for the toolbar would solve both problems.

**Comment by jnsebgosselin** [comment]
I agree that the light green and blue icons are a little bit hard to see in the light mode. I guess my screen is making things worse since it has a matte display.

**Comment by ccordoba12** [comment]
I'm going to move this one to our UX improvements repo because that's where it needs to belong.
